pkg/rewrite/rewrite.go

```diff
@@ -238,12 +241,22 @@ func Rewrite(rewriteOptions RewriteOptions) error {
 		helmMidstreams = append(helmMidstreams, *helmMidstream)
 	}
 
-	log.FinishSpinner()
-
 	if err := writeDownstreams(rewriteOptions, commonBase.GetOverlaysDir(writeBaseOptions), m, helmMidstreams, log); err != nil {
 		return errors.Wrap(err, "failed to write downstreams")
 	}
 
+	kotsKinds, err := kotsutil.LoadKotsKindsFromPath(rewriteOptions.RootDir)
+	if err != nil {
+		return errors.Wrap(err, "failed to load kotskinds")
+	}
+
+	err = store.GetStore().UpdateAppVersionInstallationSpec(rewriteOptions.AppID, rewriteOptions.AppSequence, kotsKinds.Installation)
```

Comment:
This fixes a bug.  On initial install the spec in S3 did not match the spec in PG.  The spec file on disk was updated, but the on in DB was not.